packages/faustwp-cli/utils/validateFaustEnvVars.ts

```diff
@@ -12,13 +11,8 @@ export const validateFaustEnvVars = () => {
   }
 
   if (!process.env.FAUSTWP_SECRET_KEY) {
-    warnLog('Could not find FAUSTWP_SECRET_KEY environment variable.');
-    warnLog('Some functionality may be limited.');
-  }
+    errorLog('Could not find FAUSTWP_SECRET_KEY environment variable.');
 
-  if (disableCliInteraction()) {
-    infoLog(
-      'FAUST_NO_INTERACTION is set. Specify `false`, `0`, or remove to disable.',
-    );
+    process.exit(0);
```

Comment:
We've never explicitly told people that the `FAUSTWP_SECRET_KEY` was a requirement. This would be a pretty big breaking change.

I think we should consider this a bit more before introducing.